testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/AdminClientTest.java

```diff
@@ -213,6 +217,38 @@ public void adminAuthClientDisabled() throws Exception {
         }
     }
 
+    @Test
+    public void adminAuthUserDisabled() throws Exception {
+        try (Keycloak adminClient = AdminClientUtil.createAdminClient(false, realmName, "test-user@localhost", "password", Constants.ADMIN_CLI_CLIENT_ID, null);
+             Keycloak adminClientOffline = AdminClientUtil.createAdminClient(false, ServerURLs.getAuthServerContextRoot(), realmName, "test-user@localhost", "password", Constants.ADMIN_CLI_CLIENT_ID, null, OAuth2Constants.OFFLINE_ACCESS);
+        ) {
+            // Check possible to load the realm
+            RealmRepresentation realm = adminClient.realm(realmName).toRepresentation();
+            Assert.assertEquals(realmName, realm.getRealm());
+            realm = adminClientOffline.realm(realmName).toRepresentation();
+            Assert.assertEquals(realmName, realm.getRealm());
+
+            // Disable client and check it should not be possible to load the realms anymore
+            setUserEnabled("test-user@localhost", false);
+
+            // Check not possible to invoke anymore
+            try {
+                realm = adminClient.realm(realmName).toRepresentation();
+                Assert.fail("Not expected to successfully get realm");
+            } catch (NotAuthorizedException nae) {
+                // Expected
+            }
+            try {
+                realm = adminClientOffline.realm(realmName).toRepresentation();
+                Assert.fail("Not expected to successfully get realm");
+            } catch (NotAuthorizedException nae) {
+                // Expected
+            }
```

Comment:
I prefer to use `Assert.assertThrows` for this but it's just a nit-pick. :smile:

Reply:
Ah, ok. Will try to use it for the next time.